LayoutTests/compositing/hdr/iframe-with-hdr-image.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
```

Comment:
This test has to be disabled for all platforms. Enabled for iOS and macOS WK2 only. And most likely you will need different expectation for iOS. Search for `compositing/hdr/hdr-basic-image.html` in the LayoutTest directory.